### DISCUSSION TITLE
Suggestion for best practice for MongoDB when using Caprover

### DISCUSSION THREAD

**Discussion by s-en-o** [caprover/caprover#1556] [General]
Hi all,

I have managed to get my app up and running using Caprover and MongoDB (one click app install) as the database. Also am reading [the best practice in docs](https://caprover.com/docs/best-practices.html) which is really helpful.

But I was struggle how to managed my local MongoDB instance and staging / production MongoDB instance, some of the stuff I want to do:

- Syncing
- Backup
- Restoring
- Maybe some other stuff?

I managed to do the above by logging in through SSH but I don't think that what I should do to managing my database.

Can anyone share their best practices / professional usage of Caprover with me, thanks for your time.

Cheers,
Seno

**Comment by waschinski** [comment]
I went with https://github.com/tiredofit/docker-db-backup for various database backups including a MongoDB on my CapRover instances to a remote https://github.com/minio/minio server.

**Reply by s-en-o** [reply]
Thanks @waschinski for the input, that is really helpful.
